higher_health/views.py

```diff
@@ -28,14 +27,14 @@ def healthcheck_receipt(request):
     if "saved_data" in request.session:
         data = request.session.get("saved_data")
         del request.session["saved_data"]
-        return render(request, "includes/receipt.html", data)
+        return render(request, "healthcheck_receipt.html", data)
```

Comment:
Why are we changing this and not just rendering the receipt page?

Reply:
This should be a standalone page?

Reply:
sorry I misread that completely, all good